README.md

```diff
@@ -93,6 +93,20 @@ optimism_package:
       # Prometheus docker image to use
       # Defaults to the latest image
       image: "prom/prometheus:latest"
+    # Default grafana configuration
+    grafana_params:
+      # A list of locators for grafana dashboards to be loaded be the grafana service
+      dashboard_sources: []
+      # Resource management for grafana container
+      # CPU is milicores
+      # RAM is in MB
+      min_cpu: 10
+      max_cpu: 1000
+      min_mem: 128
+      max_mem: 2048
+      # Grafana docker image to use
+      # Defaults to the latest image
+      image: "grafana/grafana:latest"
```

Comment:
is there a way we can simply *reference* the yaml file instead of having a copy of it live in this README? It seems doomed to diverge.

Reply:
this documents the default values, you don't have to include any of the values specified here in your own params file. is that what you meant?

Reply:
Ah, I had thought that this yaml content was *also* located somewhere as `default.yaml` or something like that -- is the expectation that we'll have *all* the default values as yaml content in the readme?

Reply:
there's no `default.yaml` but the default values are also present [here](https://github.com/ethpandaops/optimism-package/blob/main/src/package_io/input_parser.star). this was a pre-existing pattern so I don't actually know if that was the original intention, but it does seem to be that way

Reply:
We did have a `default.yaml` earlier to reference, however it isnt always possible to show the exhaustive config options (some conflict). Hence the README approach where we can just specify all the options and possible paths as well as plenty of examples in the `.github/tests` folder. e.g: https://github.com/ethpandaops/ethereum-package/tree/main/.github/tests